mlflow/tracking/_tracking_service/utils.py

```diff
@@ -21,6 +21,7 @@
 _TRACKING_USERNAME_ENV_VAR = "MLFLOW_TRACKING_USERNAME"
 _TRACKING_PASSWORD_ENV_VAR = "MLFLOW_TRACKING_PASSWORD"
 _TRACKING_TOKEN_ENV_VAR = "MLFLOW_TRACKING_TOKEN"
+_TRACKING_AWS_SIGV4_ENV_VAR = "MLFLOW_TRACKING_AWS_SIGV4"
```

Comment:
Instead of defining this twice, can we reference the env var defined in `mlflow.environment_variables` and call `MLFLOW_TRACKING_AWS_SIGV4.get()`?

Reply:
Done